lib/Doctrine/ORM/Mapping/MappingException.php

```diff
@@ -345,6 +345,11 @@ public static function joinColumnMustPointToMappedField($className, $joinColumn)
         ));
     }
 
+    public static function joinColumnNotAllowedOnInverseSide(string $className, string $fieldName): self
```

Comment:
This method won't be used in 2.x, will it? Let's not introduce it before 3.0, then.

Reply:
fixed